src/integrationTest/java/org/opensearch/test/framework/testplugins/dummyprotected/dummyaction/DummyAction.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.test.framework.testplugins.dummyprotected.dummyaction;
+
+import org.opensearch.action.ActionType;
+
+public class DummyAction extends ActionType<DummyResponse> {
+
+    public static final DummyAction INSTANCE = new DummyAction();
+    public static final String NAME = "cluster:admin/dummy_protected_plugin/dummy/get";
```

Comment:
nit: I assume the `NAME` is referencing to this dummy permission? so maybe rename it to `DUMMY_PERMISSION_NAME`?